megamek/src/megamek/common/IAero.java

```diff
@@ -787,5 +787,27 @@ default void useFuel(int fuelUsed) {
      */
     default void updateSensorOptions() {
     }
+    
+    // Auto Ejection Methods
+    
+    boolean isAutoEject();
+    
+    void setAutoEject(boolean autoEject);
+    
+    boolean isCondEjectAmmo();
+   
+    void setCondEjectAmmo(boolean condEjectAmmo);
+    
+    boolean isCondEjectFuel();
+    
+    void setCondEjectFuel(boolean condEjectFuel);
+    
+    boolean isCondEjectSIDest();
+    
+    void setCondEjectSIDest(boolean condEjectSIDest);
+    
+    boolean isCondEjectCTDest();
+    
+    void setCondEjectCTDest(boolean condEjectCTDest);
```

Comment:
These methods only apply to mechs and are only ever used in places that cast the entity to Mech. They don't belong in this interface.

Reply:
LAMs use this.  Even in fighter mode they don't have an SI.

Reply:
Yes, LAMs use it, and they inherit it from Mech. Aeros don't use it. In fighter mode a LAM uses its CT internal structure as SI.

Reply:
Fixed